internal/coreinternal/attraction/attraction.go

```diff
@@ -64,6 +64,11 @@ type ActionKeyValue struct {
 	// If the key has multiple values the values will be joined with `;` separator.
 	FromContext string `mapstructure:"from_context"`
 
+	// TargetType specifies the target type of an attribute to be coerced
+	// If the key doesn't exist, no action is performed.
+	// If the value cannot be coerced, the null value will be inserted.
+	TargetType string `mapstructure:"target_type"`
```

Comment:
I find this name somewhat too ambiguous. Perhaps `converted_type` or something along that could be considered as well?

Reply:
Have renamed to the action to convert, and this field to converted_type